cores/esp8266/umm_malloc/umm_malloc.c

```diff
@@ -1025,6 +1029,11 @@ void ICACHE_FLASH_ATTR *umm_info( void *ptr, int force ) {
       ++ummHeapInfo.freeEntries;
       ummHeapInfo.freeBlocks += curBlocks;
 
+#ifdef DEBUG_ESP_PORT
+      if (ummFreeSize2)
+        ummFreeSize2 += (uint64_t)curBlocks * (uint64_t)sizeof(umm_block) * (uint64_t)curBlocks * (uint64_t)sizeof(umm_block);
```

Comment:
Is it necessary for this to be a uint64_t? I think for the esp it's not necessary to use 64bit arithmetic, but as a general extension to umm it may be good. I can't tell what should be done